.evergreen/utils.sh

```diff
@@ -68,7 +68,7 @@ setup_local_atlas() {
     IMAGE=artifactory.corp.mongodb.com/dockerhub/mongodb/mongodb-atlas-local:latest
     retry podman pull $IMAGE
 
-    CONTAINER_ID=$(podman run --rm -d -e DO_NOT_TRACK=1 -P --health-cmd "/usr/local/bin/runner healthcheck" mongodb/mongodb-atlas-local:latest)
+    CONTAINER_ID=$(podman run --rm -d -e DO_NOT_TRACK=1 -P --health-cmd "/usr/local/bin/runner healthcheck" $IMAGE)
```

Comment:
awaiting discussed changes here.

Reply:
Those changes were in langchain-mongodb.  I think this one needs to stay as-is.

Reply:
Yup, you're right.